test/AutoRest.TestServerLowLevel.Tests/dpg_initial.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using AutoRest.TestServer.Tests.Infrastructure;
+using dpg_initial_LowLevel;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgInitialTest : TestServerLowLevelTestBase
+    {
+        [Test]
+        public Task DpgAddOptionalInput_NoParams_Test() => Test(async (host) =>
+        {
+            var result = await new ParamsClient(Key, host).HeadNoParamsAsync();
```

Comment:
Should we test this passing `default` for context as well?

Reply:
Added.